package.json

```diff
@@ -25,23 +25,23 @@
   "dependencies": {
     "adm-zip": "^0.5.10",
     "fancy-log": "^2.0.0",
-    "https-proxy-agent": "^5.0.1",
-    "mkdirp": "^1.0.4",
-    "node-downloader-helper": "^2.1.5",
+    "https-proxy-agent": "^7.0.1",
+    "mkdirp": "^3.0.1",
+    "node-downloader-helper": "^2.1.9",
     "progress": "^2.0.3",
-    "slugify": "^1.6.5"
+    "slugify": "^1.6.6"
   },
   "devDependencies": {
-    "@typescript-eslint/parser": "^5.47.0",
-    "chai": "^4.3.7",
-    "eslint": "^8.30.0",
+    "@typescript-eslint/parser": "^5.62.0",
+    "chai": "^4.3.8",
+    "eslint": "^8.48.0",
     "eslint-plugin-notice": "^0.9.10",
-    "mocha": "^10.1.0",
+    "mocha": "^10.2.0",
     "nyc": "^15.1.0",
-    "prettier": "^2.8.0",
-    "rimraf": "^3.0.2",
-    "sinon": "^15.0.0",
-    "typescript": "^4.9.5"
+    "prettier": "^3.0.3",
+    "rimraf": "^5.0.1",
+    "sinon": "^15.2.0",
+    "typescript": "<5.2"
```

Comment:
Is there a reason why you force TypeScript below 5.2?

Reply:
because with typescript-eslint <6 it displays a message that typescript should be <5.2

Reply:
I didn't realize that we were using TypeScript ESLint in this repo. Thanks for the clarification!